packages/server/src/index.ts

```diff
@@ -78,12 +80,25 @@ class Server implements ServerType {
 
       const enhanced_fetch: FetchApi = async (url, init?) => {
         const request_url = typeof url === 'string' ? url : url.url
-        if (request_url.startsWith('/')) {
-          // Need a smarter wau to re-enter the FAB, eventually...
-          return fetch(`http://localhost:${this.port}${request_url}`, init)
-        }
-
-        return fetch(url, init)
+        const response = await fetch(
+          request_url.startsWith('/')
+            ? // Need a smarter wau to re-enter the FAB, eventually...
+              `http://localhost:${this.port}${request_url}`
+            : url,
+          init
+        )
+        return Object.create(response, {
+          body: {
+            value: Object.create(response.body, {
+              getReader: {
+                get() {
+                  const webStream = nodeToWebStream(response.body)
```

Comment:
From what I understand, the conversion is necessary because `node-fetch` exposes node streams instead of web streams (see [their README](https://github.com/node-fetch/node-fetch#features)).

If so, I think there are broader consequences than this:
- `node-fetch` uses node streams for requests (however I am not clear and the changes that would be required)
- AWS Lambdas runs node, so we should also edit [Lambda deployer's template](https://github.com/fab-spec/fab/blob/da9f4415615718fdf2a11b39e7071786ee45dc35/packages/deployer-aws-lambda/templates/index.js#L13).

Reply:
Yeah this code definitely needs to be extracted and handled properly so both those instances pull from the same place.

To be honest, I think this all belongs inside a package like `@fab/sandbox-node-vm`. But that one was designed just to wrap the `vm` package. Realistically I need a `@fab/node-runtime` package that provides the exact `fetch` API & `Cache` that a node-hosted FAB expects...